ballerina/target/cache/tests_cache/test_suit.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
no need this file

Reply:
Fixed with a9b15196b2642d437452792bd0652bcfdaead396